Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -1837,6 +1848,19 @@ void WebProcessProxy::updateBackgroundResponsivenessTimer()
     m_backgroundResponsivenessTimer.updateState();
 }
 
+void WebProcessProxy::updateBlobRegistryPartitioningState(bool isBlobPartitioningEnabled) const
+{
+    HashSet<PAL::SessionID> alreadySentToSessions;
```

Comment:
This whole logic is not needed. A WebProcess can only have a single session. Just use `WebProcessProxy::sessionID()`.